ob.liq

```diff
@@ -8,9 +11,16 @@ set("log.file.path","/usr/local/var/log/liquidsoap/ob.log")
 set("frame.audio.channels",2)
 set("frame.audio.samplerate",48000)
 
+def onOBConnect (stream) =
+    system("#{obconnectscript} #{stream} #{status_file} #{email_state}")
+end
+def onOBDisconnect (stream) =
+    system("#{obdisconnectscript} #{stream} #{status_file} #{email_state}")
+end
+
 # Get OB Stream
 
-ob = input.http("http://icecast.ury.york.ac.uk:7070/OB-Line")
+s1 = input.http("http://#{icehost}:#{iceport}/#{obmount}", on_connect = onOBConnect("s1"), on_disconnect = onOBDisconnect("s1"))
```

Comment:
I'm assuming that's meant to be 'stream1', s1 is used for 'studio1' in quite a few places, so should probably be changed if that was the intention.